text/0000-buildpack-versions.md

```diff
@@ -0,0 +1,55 @@
+# Meta
+[meta]: #meta
+- **Name:** Semantic Version Formats for Buildpack Versions
+- **Start Date:** 2020-03-11
+- **Supersedes:** N/A
+
+# Summary
+[summary]: #summary
+
+The Buildpacks team wants to enforce format requirements for buildpack versions but will not require version semantics. This RFC explains that buildpack authors are expected to use [semantic version](https://semver.org/) format without enforcing any semantic versioning rules.
+
+# Motivation
```

Comment:
@ekcasey is it worth addding some of the benefits we get for having this format for things in `pack`, i.e. latest.

Reply:
@danielleadams is it worth mentioning here that one of the benefits of versioning in that platforms can actually make sense of version numbers based of the semantic versioning format number. i.e. "latest" is defined now.

Reply:
yeah, I can add that.